chapter_1/exercise_1_24/check_syntax.c

```diff
@@ -42,87 +39,89 @@ int getstr(char str[], int limit)
 
 void check_syntax(char str[])
 {
-  int line_nr = 1;
-
   int parentheses = 0;
   int brackets = 0;
   int braces = 0;
 
-  int single_quotes = 0;
-  int double_quotes = 0;
+  int single_quotes = OUT;
+  int double_quotes = OUT;

```

Comment:
@illustrofia I think it makes more sense to use the TRUE/FALSE approach instead of IN/OUT so that it can be more consistent with the other solutions in chapter 1.

Reply:
I have adapted the code to use the TRUE/FALSE approach.